systemtest/070-list-tags.bats

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bats
+#
+# list-tags tests
+#
+
+load helpers
+
+function setup() {
+    standard_setup
+}
+
+# list from registry
+@test "list-tags: remote repository on a registry" {
+    local remote_image=docker://quay.io/libpod/alpine_labels
+    local output=
+
+    run_skopeo list-tags --tls-verify=false $remote_image
+    expect_output --substring "quay.io/libpod/alpine_labels"
+    expect_output --substring "latest"
+}
+
+# list from a local docker-archive file
+@test "list-tags: from a docker-archive file" {
+    local file_name=./systemtest/testdata/docker-two-images.tar.xz
+    local output=
+
+    run_skopeo list-tags docker-archive:$file_name
+    expect_output --substring "example.com/empty"
```

Comment:
Is there any reason you're not checking for `:latest` and `but:different`? I don't know enough to make this a requirement, but it seems like a good idea at least superficially

Reply:
> Is there any reason you're not checking for `:latest` and `but:different`? I don't know enough to make this a requirement, but it seems like a good idea at least superficially

done